Extensions/TileMap/JsExtension.js

```diff
@@ -524,9 +524,10 @@ module.exports = {
         this._pixiObject.worldTransform.applyInverse(position, localPosition);
 
         // Check if the point is inside the object bounds
-        const originalWidth = this._pixiObject.width / this._pixiObject.scale.x;
+        const originalWidth =
```

Comment:
Instead of storing `_tilemapWidth` (and height) inside the pixi object, keep it just in pixiTileMapData and store pixiTileMapData inside `this`: `this._pixiTileMapData`.
Then you can use it to get the size whenever it's needed :)  
In the constructor, set `this._pixiTileMapData` to null.
When using it, verify if it's not null. If it is null, consider the width/height to be 0 :)

Reply:
wouldnt that just create tons of more boilerplate to keep track of? :)

Reply:
Updated now 👍 😃